shotgun_api3/lib/mockgun/mockgun.py

```diff
@@ -115,6 +115,7 @@
 """
 
 import datetime
+from typing import Any
```

Comment:
I don't like using things from outside without having a package/module name.
```suggestion
import typing
```

Then use `typing.Any`.

If you are looking for a short version, we can always aggree on
```python
import typing as tp
```

Reply:
> we cannot enforce this.

True but the first use usually defines the rule. So this is why I'd like the first iteration to mark our preferences.